Specifications/HaveAConversationWithFeelBot.feature

```diff
@@ -0,0 +1,6 @@
+Scenario : HaveAConversationWithFeelBot
+I want to have a conversation with FeelBot
+
+Given : That FeelBot is able to talk
+When : I am on my laptop or on my phone
+Then : I can have a sound conversation with FeelBot
```

Comment:
Which leaves me with a question on this way to implement features...

Some of these features are very "meta" and some other very detailed, highly redundantly so. Could you have a look regarding best practices on this topic ? Tkx